setup.py

```diff
@@ -40,4 +40,5 @@
                         'webapp2>=2.5.2',
                         'WebOb>=1.4',
                         'PyYAML>=3.11',
-                        'Werkzeug>=0.10',],)
+                        'Werkzeug>=0.10',
+                        'google-python-cloud-debugger>=1.2'],)
```

Comment:
Correct. You can use the debugger in local dev mode. The reason we added this check in Java was because Java Cloud Debugger can't work together with regular debugger (JDWP).